### PR TITLE
cypress pre-tests for Organizations tab

### PR DESCRIPTION
**Pull request by swatiDgithub**
Hello, Should I comment all the code ?

### CODE REVIEW

**Comment by l2silver** [comment]
> Hello, Should I comment all the code ?

Yeah, @swatiDgithub anything that's causing a failing test you should comment out with a #TODO comment saying to add it back in when the code for it is written

File: e2e/cypress/integration/volunteerTorontoUser/orgnizationsTab.spec.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+      beforeEach(()=>{
+        // cy.findByLabelText(inputLabels.organizations).click()                        
+        // cy.focused().should('have.attr', 'name', 'Organizations') 
+      })
+
+
+      const assertSearchBoxValidValueResults = (input)=>{
+        cy.findByLabelText(inputLabels.search).clear()
+        cy.type(input)
```
Comment (l2silver) [inline e2e/cypress/integration/volunteerTorontoUser/orgnizationsTab.spec.js:28]: Will this work? I think that you mind need to chain this to the line before (.clear().type())

File: e2e/cypress/integration/volunteerTorontoUser/orgnizationsTab.spec.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        cy.contains(input, { timeout: 500})
```
Comment (l2silver) [inline e2e/cypress/integration/volunteerTorontoUser/orgnizationsTab.spec.js:29]: So I think for this to work, what we're looking for is a row in a table with the correct user, and no other users. So we'll need to confirm that certain rows are hidden, and the correct rows are being shown. 

I believe you can use findByRole('table') to find the table, and then findByText within to find
`
const resultsTable =  cy.findByRole('table')
const correctRow = resultsTable.findByText('firstName lastName')
`

File: e2e/cypress/support/commands.js [hunk 3]
```diff
@@ -1,5 +1,12 @@
 import '@testing-library/cypress/add-commands';
```
Comment (l2silver) [inline e2e/cypress/support/commands.js:1]: Very nice!

**Review by l2silver — CHANGES_REQUESTED** [review]
I've left a few comments, but I also had another idea. 

Instead of commenting out the code, lets use ignoreTestFiles (https://docs.cypress.io/guides/references/configuration#Folders-Files) to ignore your new tests, and then we'll remove them as soon as the code is written for them.

**Comment by swatiDgithub** [comment]
Thanks Leigh. Please see latest changes. I added ignoreTestFiles, However it was not ignoring the test in my local. Will try changing the code if does not pass in build.

**Comment by swatiDgithub** [comment]
There are 2 tickets and they seem duplicate for cypress tests for Organizations Tab. 
#114 
#107 
I have referred to this feature as feature/107. Please suggest

**Review by l2silver — APPROVED** [review]
I've left a few comments, but I also had another idea. 

Instead of commenting out the code, lets use ignoreTestFiles (https://docs.cypress.io/guides/references/configuration#Folders-Files) to ignore your new tests, and then we'll remove them as soon as the code is written for them.

**Pull request merged**